Makefile_services.mk

```diff
@@ -578,96 +494,40 @@ service-ioc-si-ps-fams-stop:
 	@echo "Stopping SI PS Fams IOC services"
 	@echo "Please enter password for sirius user."
 	ansible-playbook \
-		--extra-vars "ctrl_service_state=stopped" \
-		-u sirius -k --ask-become-pass playbooks-services/playbook-service-ioc-si-ps-fams.yml
+		--extra-vars "services_to_run_corrs='' services_to_run_trims='' ctrl_service_state=stopped" \
+		-u sirius -k --ask-become-pass playbooks-services/playbook-service-ioc-si-ps.yml
 
 service-ioc-si-ps-fams-start:
 	@echo "Starting SI PS Fams IOC services"
 	@echo "Please enter password for sirius user."
 	ansible-playbook \
-		--extra-vars "ctrl_service_state=started" \
-		-u sirius -k --ask-become-pass playbooks-services/playbook-service-ioc-si-ps-fams.yml
+		--extra-vars "services_to_run_corrs='' services_to_run_trims='' ctrl_service_state=started" \
+		-u sirius -k --ask-become-pass playbooks-services/playbook-service-ioc-si-ps.yml
 
 service-ioc-si-ps-corrs-stop:
-	@echo "Stopping SI PS Correctors IOC services"
+	@echo "Stopping SI PS Corrs IOC services"
 	@echo "Please enter password for sirius user."
 	ansible-playbook \
-		--extra-vars "ctrl_service_state=stopped" \
-		-u sirius -k --ask-become-pass playbooks-services/playbook-service-ioc-si-ps-corrs.yml
+		--extra-vars "services_to_run_fams='' services_to_run_trims='' ctrl_service_state=stopped" \
+		-u sirius -k --ask-become-pass playbooks-services/playbook-service-ioc-si-ps.yml
 
 service-ioc-si-ps-corrs-start:
-	@echo "Starting SI PS Correctors IOC services"
+	@echo "Starting SI PS Corrs IOC services"
 	@echo "Please enter password for sirius user."
 	ansible-playbook \
-		--extra-vars "ctrl_service_state=started" \
-		-u sirius -k --ask-become-pass playbooks-services/playbook-service-ioc-si-ps-corrs.yml
+		--extra-vars "services_to_run_fams='' services_to_run_trims='' ctrl_service_state=started" \
+		-u sirius -k --ask-become-pass playbooks-services/playbook-service-ioc-si-ps.yml
 
 service-ioc-si-ps-trims-stop:
 	@echo "Stopping SI PS Trims IOC services"
 	@echo "Please enter password for sirius user."
 	ansible-playbook \
-		--extra-vars "ctrl_service_state=stopped" \
-		-u sirius -k --ask-become-pass playbooks-services/playbook-service-ioc-si-ps-trims.yml
+		--extra-vars "services_to_run_corrs='' services_to_run_trims='' ctrl_service_state=stopped" \
```

Comment:
shouldn't it be `--extra-vars "services_to_run_corrs='' services_to_run_fams='' ctrl_service_state=stopped" \` ?

Reply:
yes! thanks. fixed.